### ISSUE TITLE
autoprefixer configuration change

### ISSUE THREAD

**Issue by Munter** [assetgraph/assetgraph#1040]
From a build output:

```
12:27:01 PM:   Replace Autoprefixer browsers option to Browserslist config.
12:27:01 PM:   Use browserslist key in package.json or .browserslistrc file.
12:27:01 PM:   Using browsers option cause some error. Browserslist config
12:27:01 PM:   can be used for Babel, Autoprefixer, postcss-normalize and other tools.
12:27:01 PM:   If you really need to use option, rename it to overrideBrowserslist.
12:27:01 PM:   Learn more at:
12:27:01 PM:   https://github.com/browserslist/browserslist#readme
12:27:01 PM:   https://twitter.com/browserslist
```

**Comment by papandreou** [comment]
I thought that was fixed in https://github.com/assetgraph/assetgraph/commit/53d01fb0cad59c72adcd9ca526c400a348dca604, which was released in 5.10.0. Are you running an older version?

**Comment by Munter** [comment]
Ah. I probably am. 5.8.x
